zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/signal/BroadcastSignalMultiplePartitionsTest.java

```diff
@@ -121,4 +139,117 @@ public void shouldTriggerMultipleSignalCatchEvent() {
             tuple("wf_2", ProcessInstanceIntent.ELEMENT_COMPLETING),
             tuple("wf_2", ProcessInstanceIntent.ELEMENT_COMPLETED));
   }
+
+  @Test
+  public void shouldAuthorizeOnAllPartitions() {
```

Comment:
💭 Can we add one similar test case where partition 2 has two process instances, where the user has permissions to broadcast a signal to one but not the other. In that case we'd expect a one rejection and one successful correlation(?)/broadcast.